Dockerfile.web

```diff
@@ -1,6 +1,6 @@
 ARG image_tag=latest
-FROM elifesciences/journal_assets:${image_tag} AS assets
+FROM elifesciences/journal:${image_tag} AS fpm
 FROM nginx:1.13.12-alpine
 
 COPY .docker/nginx-default.conf /etc/nginx/conf.d/default.conf
-COPY --from=assets /web/ /srv/journal/web/
+COPY --from=fpm /srv/journal/web/ /srv/journal/web/
```

Comment:
The interesting part would be to see how this image should be derived from the other main `fpm` one